### PR TITLE
2 add user management

### LINKED ISSUE #2: Add user management
It should be possible to create, show, update and delete users from the admin section

### PR DESCRIPTION
**Pull request by simpros**
Added basic admin functionality

### CODE REVIEW

File: src/routes/(authenticated)/+layout.svelte [hunk 1]
```diff
@@ -19,10 +21,43 @@
 <div class="grid h-min min-h-full grid-rows-[auto_1fr]">
 	<header class="sticky top-0 z-10 bg-background bg-opacity-30 backdrop-blur-md">
 		<nav class="flex items-center justify-between p-3">
-			<a class="text-lg font-bold" href="/dash">YOS</a>
-			<form method="post" action="/api/logout">
-				<Button type="submit" size="icon" variant="ghost"><LogOut /></Button>
-			</form>
+			<a class="flex items-end gap-1 text-lg font-bold" href="/dash">
+				<enhanced:img src="../../assets/hausi.svg" class="h-10 w-10" />
```
Comment (Jan-Ka) [inline src/routes/(authenticated)/+layout.svelte]: Would be better if you'd move `hausi.svg` into a svelte component. You can use this code: https://github.com/your-own-stuff/page/blob/main/src/lib/reusables/logo.svelte

File: src/lib/create-pb-admin.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { ADMIN_EMAIL, ADMIN_PASSWORD } from '$env/static/private';
+import PocketBase from 'pocketbase';
+import type { YosPocketBase } from './types/yos-pocket-base';
+
+export async function createPbAdmin() {
+	const pbAdmin = new PocketBase('http://127.0.0.1:8090') as YosPocketBase;
```
Comment (Jan-Ka) [inline src/lib/create-pb-admin.ts]: Should read the Backend API from `.env` file

File: src/lib/create-pb-admin.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+	await pbAdmin.admins.authWithPassword(ADMIN_EMAIL, ADMIN_PASSWORD);
```
Comment (Jan-Ka) [inline src/lib/create-pb-admin.ts]: This can't be a good idea.

**Review by Jan-Ka — CHANGES_REQUESTED** [review]
Please see comments ✨

File: src/routes/(authenticated)/+layout.svelte (hunk 1 above)
Comment (simpros) [reply src/routes/(authenticated)/+layout.svelte]: Done ✔️

File: src/lib/create-pb-admin.ts (hunk 3 above)
Comment (simpros) [reply src/lib/create-pb-admin.ts]: CleanShot 2024-03-26 at 07 38 01

It is not a problem, as the pbAdmin can only be accessed on the server side - sveltekit will prevent it from building if any of the private env variables would leak into client-side code. The other option would be to update the access rights 😅

File: src/lib/create-pb-admin.ts (hunk 2 above)
Comment (simpros) [reply src/lib/create-pb-admin.ts]: Done ✔️

**Review by Jan-Ka — APPROVED** [review]
LGTM

**Pull request merged**